src/minter-suite-mapping.ts

```diff
@@ -570,17 +594,66 @@ export function handleRegistrationNFTAddresses<T>(event: T): void {
 
   if (
     event instanceof MinterHolderV0RegisteredNFTAddress ||
-    event instanceof MinterHolderV1RegisteredNFTAddress
+    event instanceof MinterHolderV1RegisteredNFTAddress ||
+    event instanceof MinterHolderV2RegisteredNFTAddress
   ) {
     handleAddManyAddressValueMinterConfig(genericEvent);
   } else if (
     event instanceof MinterHolderV0UnregisteredNFTAddress ||
-    event instanceof MinterHolderV1UnregisteredNFTAddress
+    event instanceof MinterHolderV1UnregisteredNFTAddress ||
+    event instanceof MinterHolderV2UnregisteredNFTAddress
   ) {
     handleRemoveAddressManyValueMinterConfig(genericEvent);
   }
 }
 
+export function handleMerkleV3DelegationRegistryUpdated(
+  event: MinterMerkleV3DelegationRegistryUpdated
+): void {
+  let genericEvent: ConfigValueSetAddress;
+  genericEvent = changetype<ConfigValueSetAddress>(event);
+
+  genericEvent.parameters = [
+    // dummy value ("project ID: 0") because we're not passing in a project
+    new ethereum.EventParam("_projectId", ethereum.Value.fromI32(0)),
```

Comment:
Since we're using `handleSetValueGeneric` at the minter level (and since there's no project), we had to pass in a dummy projectID here. Just wanted to call that out in case ya'll had any concerns about it
CC: @ryley-o

Reply:
SGTM! I ran with this and went a little further, changing `handleSetValueGeneric` into `handleSetMinterDetailsGeneric`since since the JSON we're updating only lives on the minter. I also moved the [type-checking](https://github.com/ArtBlocks/artblocks-subgraph/pull/135/commits/a697ac6670cda6d5e3a82f864de8064d44157acf#diff-a1077c7a5a21bf6e3a06b3124ce379bd5374fef2987d7164b250ae14fad15c56R611) to a higher scope so that we'll only update `minter.updatedAt` if the key-value's been successfully saved.